### DISCUSSION TITLE
MPV播放器可以设置用dmitrirender插帧吗

### DISCUSSION THREAD

**Discussion by ROOOCCCK** [hooke007/mpv_PlayKit#53] [Q&A 问答] [answered]
MPV播放器可以设置用dmitrirender插帧吗

**Comment by hooke007** [comment, marked as answer]
> DmitriRender is a DirectShow filter.

当然不可以

**Reply by ROOOCCCK** [reply]
为什么呢，不是很懂，因为DR可以帮我插到165，您包里面svpflow-@Starlight修改帧率到165需要怎么操作呢

**Reply by hooke007** [reply]
DirectShow filter只能用于DirectShow 播放器，其次还要适配。
而mpv甚至不是一个 DirectShow player

**Reply by hooke007** [reply]
https://github.com/hooke007/mpv_PlayKit/blob/72eae1c3d7930946146aacd39a89151556216a1b/portable_config/svpflow-%40Starlight.vpy#L11-L14
设计上是取半刷新率，要么改后边 `target_fps = max(60, fps * 2, freq)` ，要么改前面 `freq = 2*display_fps or 60`

**Reply by ROOOCCCK** [reply]
非常感谢您的讲解，我感受了MPV的魅力，调了算法，画质个人感觉跟MADVR没多大差距，而且占用低很多。就是补帧方面还是更喜欢DR那种特别流畅的效果，不知道还有没有更优秀的补帧方案呢

**Reply by hooke007** [reply]
有，等rtx4090出来了可以跑实时rife（这不是开玩笑

目前mpv没有完全gpu加速实现的低开销的补帧方案

**Reply by ROOOCCCK** [reply]
哈哈，我的3060感觉没吃够，期待更新更屌的方案

**Reply by ROOOCCCK** [reply]
请问MPV支持的缩放算法有哪些，在哪里可以查得到呢

**Reply by hooke007** [reply]
https://mpv.io/manual/master/#options-scale
https://hooke007.github.io/mpv-lazy/[06]_命令行界面.html#获取更多信息

本地手册也有

**Reply by natural-harmonia-gropius** [reply]
> 修改帧率到165需要怎么操作呢

`target_fps = freq`就好

**Reply by natural-harmonia-gropius** [reply]
> 还是更喜欢DR那种特别流畅的效果

可以试试`interpolate=quadric`那个方法，搭配svp调整比较低的target_fps看看。
svp流畅度上去伪影太严重，没试出什么好参

**Reply by ROOOCCCK** [reply]
安卓端的mpv可以配置插帧吗

**Reply by hooke007** [reply]
应该没人会用安卓压片（没有vs支持）更不要提手机那羸弱的性能（软解性能低下）了。

...硬件实现： 高通8系有自带的memc，通常只支持适配的视频播放器（看厂商）
半软件实现有暂且免费的svplayer

**Reply by ROOOCCCK** [reply]
我是865，有memc适配的相关播放器吗

**Reply by hooke007** [reply]
> （看厂商）

这里指的就是 问终端制造商

**Reply by natural-harmonia-gropius** [reply]
> 我是865，有memc适配的相关播放器吗

理论上，手机自带相册可以
